rancid-tomatillos/src/containers/App/App.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './App.css';
```

Comment:
Once we change App.css to App.scss we will need to change the file name here as well.